satellite/analytics/service.go

```diff
@@ -101,6 +120,58 @@ func (service *Service) enqueueMessage(message segment.Message) {
 	}
 }
 
+func (service *Service) createHubSpotUser(data PayloadUser) {
+
+	payloadBytes, err := json.Marshal(data)
+	if err != nil {
+		service.log.Error("Error message", zap.Error(err))
+	}
+
+	body := bytes.NewReader(payloadBytes)
+
+	url := "https://api.hubapi.com/crm/v3/objects/contacts?hapikey=" + service.config.HubSpotApiKey
```

Comment:
use proper escaping, also ensure that tests won't hit the live server

Reply:
Do you mean do something like `"https://api.hubapi.com/..../contacts?hapikey=" + url.QueryEscape(service.config.HubspotApiKey)`?